packages/flutter_tools/test/web/devices_test.dart

```diff
@@ -26,13 +26,27 @@ void main() {
         return null;
       });
     });
+
+    test('Defaults', () async {
+      final ChromeDevice device = ChromeDevice();
+
+      expect(device.name, 'Chrome');
+      expect(device.id, 'chrome');
+      expect(device.supportsHotReload, true);
+      expect(device.supportsHotRestart, true);
+      expect(device.supportsStartPaused, true);
+      expect(device.supportsFlutterExit, true);
+      expect(device.supportsScreenshot, false);
+      expect(await device.isLocalEmulator, false);
+    });
+
     testUsingContext('Invokes version command on non-Windows platforms', () async{
       when(mockPlatform.isWindows).thenReturn(false);
       when(mockProcessManager.canRun('chrome.foo')).thenReturn(true);
       when(mockProcessManager.run(<String>['chrome.foo', '--version'])).thenAnswer((Invocation invocation) async {
         return MockProcessResult(0, 'ABC');
       });
-      final WebDevice webDevice = WebDevice();
+      final ChromeDevice webDevice = ChromeDevice();
```

Comment:
Shouldn't `webDevice` variable be renamed to `chromeDevice` too? (For consistency reasons)

Reply:
Good point, fixed